tasks/setup-Debian.yml

```diff
@@ -1,13 +1,43 @@
 ---
+- name: Ensure APT HTTPS Transport is installed.
+  apt:
+    name: apt-transport-https
+    state: installed
+  when: ansible_distribution_release != "xenial"
```

Comment:
Xenial is excluded because the role uses the Varnish package available on Ubuntu's repo, so apt-transport-https it's not needed in this case.